packages/agoric-cli/src/commands/inter.js

```diff
@@ -64,34 +65,38 @@ const makeFormatters = assets => {
     const [dl, dm] = fmtAmtTuple(br(r.denominator));
     return `${Number(nm) / Number(dm)} ${nl}/${dl}`;
   };
+  /** @param {Ratio} r */
   const discount = r =>
     100 - (Number(r.numerator.value) / Number(r.denominator.value)) * 100;
-  return { amount, record, price, discount };
-};
 
-/**
- * Format amounts in vaultManager metrics for JSON output.
- *
- * @param {*} metrics manager0.metrics
- * @param {*} quote manager0.quote
- * @param {*} assets agoricNames.vbankAssets
- */
-const fmtMetrics = (metrics, quote, assets) => {
-  const fmt = makeFormatters(assets);
-  const { liquidatingCollateral, liquidatingDebt } = metrics;
+  /** @param {import('@agoric/time/src/types.js').TimestampRecord} tr */
+  const absTime = tr => new Date(Number(tr.absValue) * 1000).toISOString();
+  /** @param {import('@agoric/time/src/types.js').RelativeTimeRecord} tr */
+  const relTime = tr =>
+    new Date(Number(tr.relValue) * 1000).toISOString().slice(11, 19);
 
-  const {
-    quoteAmount: {
-      value: [{ amountIn, amountOut }],
-    },
-  } = quote;
-  const price = fmt.price({ numerator: amountOut, denominator: amountIn });
-
-  const amounts = objectMap(
-    { liquidatingCollateral, liquidatingDebt },
-    fmt.amount,
-  );
-  return { ...amounts, price };
+  /** @param {bigint} n */
```

Comment:
took me a minute without context to see what this was doing
```suggestion
  /** @param {bigint} bp */
```